ansible/roles/setup_desktop/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+---
+# Desktop environment specific packages and configuration
+- name: Add ucaresystem-core repository
+  ansible.builtin.apt_repository:
+    repo: "ppa:utappia/stable"
+    state: present
+
+- name: Add Helm APT repository key
+  ansible.builtin.apt_key:
+    url: "https://baltocdn.com/helm/signing.asc"
+    state: present
+
+- name: Add Helm APT repository
+  ansible.builtin.apt_repository:
+    repo: "deb https://baltocdn.com/helm/stable/debian/ all main"
+    state: present
```

Comment:
This is related to development section , apt_key and repo should be declared in group_vars, not hardcoded